llvm/lib/CodeGen/IntrinsicLowering.cpp

```diff
@@ -312,6 +312,12 @@ void IntrinsicLowering::LowerIntrinsicCall(CallInst *CI) {
     CI->replaceAllUsesWith(ConstantInt::get(Type::getInt64Ty(Context), 0));
     break;
   }
+  case Intrinsic::readfixedtimer: {
+    errs() << "WARNING: this target does not support the llvm.readfixedtimer"
+           << " intrinsic.  It is being lowered to a constant 0\n";
+    CI->replaceAllUsesWith(ConstantInt::get(Type::getInt64Ty(Context), 0));
```

Comment:
`Constant::getNullValue(Type::getInt64Ty(Context))`?

Reply:
I copied it from the other one, but either way works. I think setting it to zero is a little more explicit.